lib/Staticman.js

```diff
@@ -488,4 +488,21 @@ Staticman.prototype.setUserAgent = function (userAgent) {
   this.userAgent = userAgent
 }
 
+Staticman.prototype.getSiteConfig = function(force) {
+  return new Promise((resolve, reject) => {
+    this
```

Comment:
When chaining functions across multiple lines, I'd rather see the first one in the same line as the object being called on. So:

```js
staticman.getSiteConfig()
  .then()
  .catch()
```

Would you mind changing that?